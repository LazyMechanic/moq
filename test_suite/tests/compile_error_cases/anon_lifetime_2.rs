struct DummyWrapper<'a>(&'a ());

#[moq::automock]
trait Trait {
    fn f(&self, arg: DummyWrapper<'_>);
}

fn main() {}
