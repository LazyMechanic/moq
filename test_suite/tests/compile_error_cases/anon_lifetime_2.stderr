error: anonymized lifetimes not supported
 --> tests/compile_error_cases/anon_lifetime_2.rs:5:35
  |
5 |     fn f(&self, arg: DummyWrapper<'_>);
  |                                   ^^
